src/commands/handler.js

```diff
@@ -5,39 +5,32 @@ const help = require('./help');
 const bill = require('./bill/handler');
 const config = require('./config/handler');
 
-const permissions = require('./permissions.json');
+const database = require('./../database');
 
 /**
  * Checks if the message object's user has the Role necessary to perform the command
  * return true if it has permission, false otherwise.
  *
- * The user must have at least one of the string in the `permArray` as his role in order to get true
- * if the permArray is undefined or contains 0 itens it will allow the command.
+ * The user must have at least one of the string in the `requiredRole` as his role in order to get true
+ * if the requiredRole is undefined or contains 0 itens it will allow the command.
  *
  * @param {object} message
- * @param {string[]} permArray
+ * @param {string} desiredCommand
  * @returns {boolean}
  */
-const canDoCommand = (message, permArray) => {
-  if (permArray && Array.isArray(permArray)) {
-    if (permArray.length === 0) {
+const canDoCommand = (message, desiredCommand) => {
+  const requiredRole = database.getPermission(desiredCommand);
```

Comment:
not sure if the database should need another method (different than getConfig) but this could be changed later

Reply:
It needs this because the `setConfig` method writes only to the `config` property of the object. this can lead to a config colliding with a command name. So I've created a special method that writes to other property.